playbilling/README.md

```diff
@@ -0,0 +1,64 @@
+# Play Billing
+
+The Play Billing module provides capabilities for your TWA app to connect with [Google Play Billing library](https://developer.android.com/google/play/billing), for example you can:
+
+* Query Purchases history
+* Initialize a payment
+* Query SKU details (to be moved to ProductDetails later)
+
+The module uses [Version 5](https://developer.android.com/google/play/billing/release-notes#5-2-1)  of Play Billing library.
+
+
+## How to use it? (Website)
+To use this data from your website you can do it as follows:
+```
```

Comment:
You can add the language to Markdown codeblocks to make syntax highlighting better - so here add js after the three backticks (and below add xml and java).